### PR TITLE
REOPEN: REVERT: Changes faster fireman carrying from a trait on gloves to a skillchip

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

image

reopenage of #58239, which is a reopenage of #58224 

## Why It's Good For The Game

image

## Changelog
:cl:
balance: Changes the faster fireman carrying traits on skillchips into a nitrile glove trait.
/:cl:

### CODE REVIEW

**Comment by tralezab** [comment]
see my commit for my changes, basically i did the review and removed the nanobot lore as it was really stupid. also simplified the pickup message to be clearer

**Comment by tralezab** [comment]
sorry for burying conversations on the last pr by the way, really my bad on pring the wrong branch

**Comment by Qustinnus** [comment]
bigger brain: make insults a skill chip

**Comment by Colovorat** [comment]
Paratiders nerf in some way. I don't like it. Bad

**Comment by SuperNovaa41** [comment]
> Paratiders nerf in some way. I don't like it. Bad

How is nerfing people refusing to do their job a bad thing

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -919,19 +919,19 @@
...
 	else if(HAS_TRAIT(src, TRAIT_QUICK_CARRY))
 		carrydelay = 4 SECONDS
 		skills_space = " quickly"
 
-	visible_message("<span class='notice'>[src] starts[skills_space] lifting [target] onto [p_their()] back..</span>",
+	visible_message("<span class='notice'>[src] starts[skills_space] lifting [target] onto [p_their()] back...</span>",
 	//Joe Medic starts quickly/expertly lifting Grey Tider onto their back..
-	"<span class='notice'>[carrydelay < 3.5 SECONDS ? "Using your fireman carrying training, you" : "You"][skills_space] start to lift [target] onto your back[carrydelay == 4 SECONDS ? ", with ease thanks to your advanced knowledge.." : "..."]</span>")
-	//(Using your fireman carrying training, you/You) ( /quickly/expertly) start to lift Grey Tider onto your back(, with ease thanks to your advanced knowledge../...)
+	"<span class='notice'>You[skills_space] start to lift [target] onto your back...</span>")
```
Comment (Mothblocks) [inline code/modules/mob/living/carbon/human/human.dm:933]: Indent up.

Comment (Mothblocks) [reply code/modules/mob/living/carbon/human/human.dm:933]: The line above too.

**Pull request merged**